### PR TITLE
Abstract revisions

### PR DESCRIPTION
**Pull request by zietzm**
I tried to bring the point that, "Performance attributable to factors other than degree is often only a small portion of overall performance," to a more prominent position. 

Do these revisions make the abstract too results-first? I'm still not certain that line 9 belongs in the abstract, as important of a point as I think it is.

### CODE REVIEW

File: content/01.abstract.md [hunk 1]
```diff
@@ -5,10 +5,10 @@ The number of edges connecting to a node, termed degree, can vary greatly across
 If degree strongly influences edge prediction, then imbalance or bias in the distribution of degrees could lead to nonspecific or misleading predictions.
 We introduce a network permutation framework to quantify the effects of node degree on edge prediction.
 Our framework decomposes performance into the proportions attributable to degree and the network's specific connections.
-Additionally, our permutation-derived edge prior quantifies the probability that an edge exists based only on node degree.
+We discovered that performance attributable to factors other than degree is often only a small portion of overall performance.
```
Comment (dhimmel) [inline content/01.abstract.md]:
```suggestion
We discover that performance attributable to factors other than degree is often only a small portion of overall performance.
```

File: content/01.abstract.md [hunk 2]
```diff
@@ -5,10 +5,10 @@ The number of edges connecting to a node, termed degree, can vary greatly across
...
+We introduce the permutation-derived edge prior as the probability that an edge exists based only on degree.
 The edge prior shows excellent discrimination and calibration for 20 biomedical networks (16 bipartite, 3 undirected, 1 directed), with AUROCs frequently exceeding 0.85.
-Feature performance exceeding the edge prior is attributable to factors other than degree, and we find that it is often only a small portion of overall performance.
-Degree's predictive performance diminishes when the networks used for training and testing, despite measuring the same biological relationships, were generated using distinct techniques and hence have large differences in degree distribution.
 Researchers seeking to predict new or missing edges in biological networks should use the edge prior as a baseline.
-Degree preserving permutation allows researchers to directly investigate which features used for prediction capture degree and which also capture additional information.
+Degree-preserving permutation allows researchers to investigate directly which edge-prediction features capture degree and which also capture additional information.
```
Comment (dhimmel) [inline content/01.abstract.md]: Is this sentence redundant with:

> Our framework decomposes performance into the proportions attributable to degree and the network's specific connections.

I think ti does add that the permutations are degree preserving, but that could be achieved when permutation is mentioned?

Comment (zietzm) [reply content/01.abstract.md]: I think you are correct that it is redundant. What if we just remove the sentence?

File: content/01.abstract.md [hunk 3]
```diff
@@ -5,10 +5,9 @@ The number of edges connecting to a node, termed degree, can vary greatly across
...
+We introduce the permutation-derived edge prior as the probability that an edge exists based only on degree.
 The edge prior shows excellent discrimination and calibration for 20 biomedical networks (16 bipartite, 3 undirected, 1 directed), with AUROCs frequently exceeding 0.85.
-Feature performance exceeding the edge prior is attributable to factors other than degree, and we find that it is often only a small portion of overall performance.
-Degree's predictive performance diminishes when the networks used for training and testing, despite measuring the same biological relationships, were generated using distinct techniques and hence have large differences in degree distribution.
-Researchers seeking to predict new or missing edges in biological networks should use the edge prior as a baseline.
-Degree preserving permutation allows researchers to directly investigate which features used for prediction capture degree and which also capture additional information.
+Researchers seeking to predict new or missing edges in biological networks should use the edge prior as a baseline to identify the fraction of performance that is nonspecific because of degree.
```
Comment (dhimmel) [inline content/01.abstract.md:12]: "to identify the fraction of performance that is nonspecific because of degree" is a great summary IMO

**Review by dhimmel — APPROVED** [review]

**Pull request merged**